### ISSUE TITLE
[ReDash] Data Source of type MySQL is missing into the list.

### ISSUE THREAD

**Issue by damusthe** [bitnami/vms#1534]
### Platform

Virtual Machine

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

edc728f0-d388-4537-f365-5fb752e45703

### bndiagnostic was not useful. Could you please tell us why?

Not related to my issue

### Describe your issue as much as you can

I downloaded the Bitnami ReDash v10.1.0-r97 from [https://bitnami.com/stack/redash/virtual-machine](https://bitnami.com/stack/redash/virtual-machine) and VM starts fine into VirtualBox.
But I can't create a new Data Source of type **MySQL**, because it's missing into the list.

See also #744

**Comment by jotamartos** [comment]
As mentioned in the other thread, we take the reDash files from [its GitHub repository](https://github.com/getredash/redash/tags) and configure it for you to use it. If data sources do not appear there, it may be related to an issue in the application or with the way they have to enable the different data sources depending on if you are hosting the application or not. Can you ask the app's developers to know more about this?

**Comment by damusthe** [comment]
@jotamartos I'll try to post on ReDash GittHub.

**Comment by jotamartos** [comment]
Hi all, 

> My best guess is that some of the environment changes the Bitnami team made has led to some Python dependencies getting lost, and the query runners are failing.

That was probably the reason behind the issue but I'm afraid to tell you that we just deprecated support for Redash in our catalog. The Bitnami solution is no longer maintained and it was removed from the cloud marketplaces as well. 

In the future, if you want to continue using the application, you will need to migrate your data to the solution that the upstream project provides or [any other alternative solution](https://bitnami.com/stacks/analytics).

Sorry for the inconvenience.
